### DISCUSSION TITLE
Column Hiding - Accessing

### DISCUSSION THREAD

**Discussion by brooksa-chemtest** [TanStack/table#4893] [Q&A]
Hi there

I'm new to this and have a question I can't seem to find the answer to.

I am passing an array of objects into a table. 2 values in each object will be used to create a link to a new page that is placed in the final cell of the row.

If I display both values in my table, I can access both of them and create my link as expected, but is there a way to achieve this without displaying the 2 values as columns?

I have found a way to hide the columns from the Table component I have created, but I want to reuse this across my application (passing in the required column values), so was hoping I could store the logic in the columns I have defined.

Is this possible?

**Comment by jschuur** [comment]
Sounds like we have a [very similar](https://github.com/TanStack/table/discussions/4106#discussioncomment-6996996) question.
